conf/airframes/examples/bixler_lisa_m_2.xml

```diff
@@ -81,6 +85,33 @@
     <define name="BODY_TO_IMU_PHI"   value="0." unit="deg"/>
     <define name="BODY_TO_IMU_THETA" value="0." unit="deg"/>
     <define name="BODY_TO_IMU_PSI"   value="0." unit="deg"/>
+    
+    <!-- Dummy Gyro values for NPS -
```

Comment:
Doesn't really make sense to put "dummy" values for GYRO and ACCEL here... just omit them and the defaults for the IMU are used and you should be actually able to fly with them...